components/inputs/input-date.js

```diff
@@ -277,14 +287,15 @@ class InputDate extends FocusMixin(LabelledMixin(SkeletonMixin(FormElementMixin(
 				<div>${shortDateFormat}</div>
 			</div>
 			${errorTooltip}
+			${revertTooltip}
 			<d2l-dropdown ?disabled="${this.disabled || this.skeleton}" no-auto-open ?prefer-fixed-positioning="${this.preferFixedPositioning}">
 				<d2l-input-text
 					?novalidate="${this.noValidate}"
 					aria-invalid="${this.invalid ? 'true' : 'false'}"
 					@blur="${this._handleInputTextBlur}"
 					@change="${this._handleChange}"
 					class="d2l-dropdown-opener vdiff-target"
-					instructions="${ifDefined((this._showInfoTooltip && !errorTooltip && !this.invalid && this.childErrors.size === 0 && this._inputTextFocusShowTooltip) ? this.localize(`${this._namespace}.openInstructions`, { format: shortDateFormat }) : undefined)}"
+					instructions="${ifDefined((this._showInfoTooltip && !errorTooltip && !revertTooltip && !this.invalid && this.childErrors.size === 0 && this._inputTextFocusShowTooltip) ? instructionsMessage : undefined)}"
```

Comment:
Jeff's design has a newline after "[field] has been reverted". This is possible to do in revert tooltip if I separate the lang terms and wrap them in divs in the `d2l-tooltip` but it seems this might not be possible for when we pass the string into instructions. Let me know if it is. I tried a variety of newline implementations but maybe didn't get the right one.

Otherwise we might need to have the text update in revert tooltip instead (though would need to test if it re-announces on change).